apps/src/templates/CensusForm.jsx

```diff
@@ -41,15 +42,36 @@ const styles = {
 class CensusForm extends Component {
 
   state = {
-    value: ''
+    submission: {
+      name: '',
+      email: '',
+      role: '',
+      pledge: false,
+    }
   };
 
-  handleChange = event => {
-    this.setState({value: event.target.value});
+  handleChange = (propertyName, event) => {
+    const submission = this.state.submission;
+    submission[propertyName] = event.target.value;
```

Comment:
This is mutating state, which we don't want to do in React.

Reply:
I think you could do something like
```
this.setState({
  submission: {
    ...this.state.submission,
    [propertyName]: event.target.value
  }
});
```

Reply:
Thanks for pointing out the state mutation Brent. I was going to recommend this syntax, but I couldn't remember why it was important. 🤔